resources/conf/config.toml.template

```diff
@@ -189,6 +189,20 @@ retainKeys = ["self_validate_jwt", "issuer", "claim_mappings", "consumer_key_cla
   # Max interval for the Envoy's exponential retry back off algorithm
   maxInterval = "500ms"
 
+# TCP configurations applicable with the upstream clusters
+[router.upstream.tCPConfigurations]
+  # The number of milliseconds a connection needs to be idle before keep-alive probes start being sent
+  keepaliveTimeInMillis = 120000
+  # Maximum number of keep-alive probes to send without response before deciding the connection is dead
+  keepaliveProbes = 9
+  # The number of milliseconds between keep-alive probes
+  keepaliveIntervalInMillis = 75000
+
+# HTTP configurations applicable with the upstream clusters
+[router.upstream.hTTPConfigurations]
+  # Idle timeout in milliseconds for connections
+  idleTimeoutInMillis = 120000
```

Comment:
This is too low for default. We have to set a value which is more than max response timeout (300s)
```suggestion
  idleTimeoutInMillis = 360000
```